docs/getting-started/08-connect-external-application.md

```diff
@@ -0,0 +1,278 @@
+      example: orders-service
+  EOF
+  ```
+
+2. Check that the Application CR was created. Its phase should state `deployed`:
+
+   ```bash
+   kubectl get application commerce-mock -o=jsonpath="{.status.installationStatus.status}"
+   ```
+3. Get a token required to connect Commerce mock to the Application CR. To do that, create the [TokenRequest CR](/components/application-connector/#custom-resource-token-request). The CR name must match the name of the application for which you want to get the configuration details. Run this command:
+
+  ```yaml
+  cat <<EOF | kubectl apply -f -
+  apiVersion: applicationconnector.kyma-project.io/v1alpha1
+  kind: TokenRequest
+  metadata:
+    name: commerce-mock
+  EOF
+  ```
+
+4. Fetch the TokenRequest CR you created to get the token from the status section:
+
+   ```bash
+   kubectl get tokenrequest commerce-mock -o=jsonpath="{.status.url}"
+   ```
+   >**NOTE:** If the response doesn't contain any content, wait for some time and run the command again.
+
+   The system returns a response similar to this one:
+
+   ```bash
+   https://connector-service.{CLUSTER_DOMAIN}/v1/applications/signingRequests/info?token=h31IwJiLNjnbqIwTPnzLuNmFYsCZeUtVbUvYL2hVNh6kOqFlW9zkHnzxYFCpCExBZ_voGzUo6IVS_ExlZd4muQ==
+   ```
+
+   Save this output with the token to the clipboard, as you will need it in the next steps.
+
+   > **CAUTION:** The token included in the output is valid for 5 minutes.
+
+  </details>
+  <details>
+  <summary label="console-ui">
+  Console UI
+  </summary>
+
+1. Return to the general view in the Console UI by selecting **Back to Namespaces**.
+2. Go to **Integration** > **Applications/Systems** and select **Create Application**.
+3. Set the application's name to `commerce-mock` and select **Create** to confirm the changes.
+
+  Wait for the application to have the `SERVING` status.
+
+4. Open the newly created application and select **Connect Application**.
+5. Copy the whole URL string with the token by clicking **Copy to Clipboard** and select **OK** to close the pop-up box.
+
+  > **CAUTION:** The token included in the URL is valid for 5 minutes.
+
+</details>
+</div>
+
+### Connect events
+
+To connect events from Commerce mock to the microservice, follow these steps:  
+
+1. Once in the `order-service` Namespace, go to **Configuration** > **API Rules** > **commerce-mock** in the left navigation panel.
+2. Open the link under the **Host** column to access Commerce mock.
+3. Click **Connect**.
+4. Paste the previously copied URL string with the token. Confirm by selecting **Connect** and wait until the application gets connected.
+5. Select **Register All** on the **Local APIs** tab or just register **SAP Commerce Cloud - Events** to be able to send events.
+
+Once registered, you will see all Commerce mock APIs and events available under the **Remote APIs** tab.
+
+> **TIP:** Local APIs are the ones available within the mock application. Remote APIs represent the ones registered in Kyma.
```

Comment:
These two sentences confuse me a little as to which are the representations (representer) and which are the actual APIs (representee/represented). 
I feel like the first sentence says that Local APIs are represented (...), while the second one says that Remote APIs represent (...).
I believe they should be balanced equally, one way or another.
```suggestion
> **TIP:** Local APIs are the ones available within the mock application. Remote APIs are the ones registered in Kyma.
```